ZenPacks/zenoss/WSMAN/datasources/WSMANDataSource.py

```diff
@@ -247,15 +257,12 @@ def onSuccess(self, results, config):
             result_timestamp_key = datasource.params.get(
                 'result_timestamp_key')
 
-            timestamp = None
+            timestamp = 'N'
```

Comment:
Why set timestamp twice in the case that we have a result_timestamp_key? It would seem better to just put `timestamp = 'N'` in an `else` clause of the `if` statement below.